artman/cli/__init__.py

```diff
@@ -16,6 +16,7 @@
 
 from artman.cli import configure
 from artman.cli import main
+from artman.cli import main2
 from artman.cli import support
 
-__all__ = ('configure', 'main', 'support')
+__all__ = ('configure', 'main', 'main2' 'support')
```

Comment:
Comma between `'main2'` and `'support'`. As it stands this evaluates to `'main2support'`.